databaker/framework.py

```diff
@@ -35,6 +37,15 @@ def loadxlstabs(input, sheetids="*", verbose=True):
             tableset = XLSXTableSet(filename=input_file_name, fileobj=input_file_obj)
         elif str(input_file_name).endswith(".xls"):
             tableset = XLSTableSet(filename=input_file_name, fileobj=input_file_obj)
+        elif str(input_file_name).endswith(".ods"):
+            df_dict = pd.read_excel(str(input_file_name), engine="odf", sheet_name=None)
```

Comment:
need to check if this works with a file object, so if I do:

```
with open("myfile.ods", "r") as f:
        loadxlstabs(f)
```

it needs to support this (this is principally how data will be coming in from gssutils so we need to know). I've added a comment to the feature for testing this.

Reply:
Added a test which uses this as a template for the step definition - leads to `context.tabs = ` and then follows the same structure as existing tests.